src/detect-threshold.c

```diff
@@ -177,6 +177,10 @@ static DetectThresholdData *DetectThresholdParse(const char *rawstr)
             de->track = TRACK_DST;
         if (strncasecmp(args[i],"by_src",strlen("by_src")) == 0)
             de->track = TRACK_SRC;
+        if (strncasecmp(args[i],"by_both",strlen("by_both")) == 0)
```

Comment:
I suppose this should actually not use strncasecmp, but simply strcasecmp. Not an issue you introduced here clearly

Reply:
Is this something you want me to change in the other cases as well?